qstash/api/topics/remove-endpoint.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Remove Endpoints"
+description: "Remove one or more endpoints"
+api: "DELETE /v2/topics/{topicName}/endpoints"
+authMethod: "bearer"
+---
+
+Remove one or multiple endpoints from a topic. If all endpoints have been removed, the topic will be deleted.
+
+
+## Request
+
+<ParamField path="topicName" type="string" required>
+  The name of your topic. If it doesn't exist yet, it will be created.
```

Comment:
Here if the topic does not exist, we return error.
```
 errors.NotFoundError{ResourceType: "topic", ResourceId: topicName}
 ```

Reply:
sorry, copy paste mistake